daemons/execd/execd_commands.c

```diff
@@ -942,11 +942,11 @@ action_complete(svc_action_t * action)
         int timeout_left = cmd->timeout_orig - time_sum;
         int delay = cmd->timeout_orig / 10;
 
-        if(delay >= timeout_left && timeout_left > 20) {
-            delay = timeout_left/2;
+        if ((delay >= timeout_left) && (timeout_left > 20)) {
+            delay = timeout_left / 2;
         }
+        delay = QB_MAX(2000, delay); // Use a minimum 2s delay
```

Comment:
Hmm I misunderstood the problem. The real issue is that when multiple systemd resources are starting in parallel, systemd will still start them one by one, causing the later ones to show as "inactive" (rather than "pending") until the earlier ones start. See https://bugs.clusterlabs.org/show_bug.cgi?id=5543

I'll drop this commit, I think this will need the full fix in T25

Reply:
Yep that was clear that this wasn't a real fix. Just something that wouldn't introduce noticeable delay and raise the chance the resource was already starting/stopping when checked.